American-Elections-and-Crypto.md

```diff
@@ -0,0 +1,48 @@
+|   |   |
+|---|---|
+| Title               |  Questioning the Future: American Elections and Crypto from a Cypherpunk Perspective |
+| Author              | João Paulo Sant'Anna da Silva |
+| Featured text       | FIXME |
+| CT link             | FIXME |
+| CT date             | YYYY-MM-DD FIXME |
+| CT tweet            | FIXME |
+| YouTube link        | FIXME |
+| YouTube tweet       | FIXME |
+| Twitter full video  | FIXME |
+| Twitter clip 1      | FIXME |
+| Twitter clip 2      | FIXME |
+
+
+Thesis statement - What are the implications of the American election to crypto? will they change something?
+
```

Comment:
Nice topic. I wonder would it be possible to do a section on the last 2 elections and see what happened with crypto at the times around the elections and any correlations

Reply:
Thanks, i will try to add how the market behaved on last elections. It may be a good add to the article